engine/src/test/java/io/zeebe/engine/state/deployment/WorkflowStateTest.java

```diff
@@ -506,8 +506,8 @@ public void shouldNotGetWorkflowsWithOtherProcessId() {
   public void shouldReturnHighestVersionInsteadOfMostRecent() {
     // given
     final String processId = "process";
-    workflowState.putDeployment(creatingDeploymentRecord(zeebeState, processId, 2));
     workflowState.putDeployment(creatingDeploymentRecord(zeebeState, processId, 1));
+    workflowState.putDeployment(creatingDeploymentRecord(zeebeState, processId, 2));
```

Comment:
Changing the order of the statements doesn't align with the intention of the test case 
` shouldReturnHighestVersionInsteadOfMostRecent`.

Reply:
yeah sorry was a leftover - I first tried to prevent to have smaller versions after a higher is deployed